qdrant-landing/content/documentation/frameworks/spring-ai.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Spring AI
+weight: 2200
+---
+
+# Spring AI
+
+[Spring AI](https://docs.spring.io/spring-ai/reference/) is a Java framework that provides a [Spring-friendly](https://spring.io/) API and abstractions for developing AI applications.
+
+Qdrant is available as supported vector database for use within your Spring AI projects.
+
+## Installation
+
+To acquire Spring AI artifacts, declare the Spring Snapshot repository in your `pom.xml`.
+
+```xml
+<repository>
+	<id>spring-snapshots</id>
+	<name>Spring Snapshots</name>
+	<url>https://repo.spring.io/snapshot</url>
+	<releases>
+		<enabled>false</enabled>
+	</releases>
+</repository>
+```
+
+Add the `spring-ai-qdrant` package.
+
+```xml
+<dependency>
+	<groupId>org.springframework.ai</groupId>
+	<artifactId>spring-ai-qdrant</artifactId>
+	<version>
+		<!-- VERSION -->
+	</version>
+</dependency>
+```
+
+## Usage
+
+You can set up the Qdrant vector store with the `QdrantVectorStoreConfig` options.
+
+```java
```

Comment:
If I'm reading your Spring PR correctly, should you make this code block consistent with [this one](https://github.com/spring-projects/spring-ai/pull/345/files#diff-f60394d64214cf74d15f594e7568e103290e0052640d3e0af2e1f98cc701fa98R85-R94)?